### ISSUE TITLE
Wrong version of node in .nvmrc file

### ISSUE THREAD

**Issue by annimorphism** [WebThingsIO/gateway#2873]
In the readme, the node versions show 12.19.0
  ```
  $ node --version
  v12.19.0
  $ npm --version
  6.14.8
  ```
  But in the .nvmrc file, the default version is lts/dubnium. I am unable to install the gateway on my system with dubnium but it works smoothly with 12.19.0. Am I doing something wrong here?
  
  My system: Macbook Pro 2015 with Big Sur

**Comment by Utopiah** [comment]
Indeed, it's also not coherent with `FROM node:12-buster-slim in https://github.com/WebThingsIO/gateway/blob/master/Dockerfile#L1

**Comment by benfrancis** [comment]
@annimorphism Short answer, you can the latest LTS version of Node.js (v14):
```
$ nvm install --lts
$ nvm use --lts
```

Longer answer... 

I see that @dyap2 updated the version in the README to v12 in https://github.com/WebThingsIO/gateway/pull/2601 but the version in .nvmrc is still v10 (dubnium).

Node v14 has been supported since 1.0.0 as noted in the [release notes](https://github.com/WebThingsIO/gateway/releases/tag/1.0.0). I'm not exactly sure why .nvmrc is still at 10, but I suspect it's because some of the builds (Raspbian, Docker, .deb, .rpm etc.) are still using it.

We do need to upgrade everything from v10 to either v12 or v14, ideally in the next release, because as I understand it v10 stopped receiving security updates in April.

I see from @Utopiah's comment above that the Docker image appears to be using v12. My main concern is whether the current version of Raspbian/Raspberry Pi OS the base image is using supports the latest LTS version of Node, because we currently have no way to automatically upgrade users to a newer version (see #2801).

**Comment by Utopiah** [comment]
Word of caution that this is for the gateway itself, add-ons might need to be re-installed and might not work after so probably safer to do that node version change while keeping an eye on the logs.

**Comment by benfrancis** [comment]
> My main concern is whether the current version of Raspbian/Raspberry Pi OS the base image is using supports the latest LTS version of Node, because we currently have no way to automatically upgrade users to a newer version

I note that the version of Node.js provided in the Raspbian Buster repo is v10, but since we install it using nvm in the build scripts it should be possible to specify a newer version https://github.com/WebThingsIO/gateway/blob/master/image/stage3/01-install-deps/01-run-chroot.sh#L12
